spec/lib/validators/datetime_setting_validator_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+RSpec.describe DatetimeSettingValidator do
+  describe "#valid_value?" do
+    subject(:validator) { described_class.new }
+
+    it "returns true for blank values" do
```

Comment:
at this point, we're basically testing how `DateTime.iso8601(value)` works. I would just test blank values, one valid value with Z, one valid value with T, and one invalid value